README.rst

```diff
@@ -1637,6 +1697,23 @@ List device groups
 
     GET /api/v1/controller/group/
 
+**Available filters**
+
+You can filter a list of device groups based on 
+their organization using the ``organization_id``.
+
+.. code-block:: text
+
+   GET /api/v1/controller/group/?organization={organization_id}
+
+You can filter a list of device groups that have a 
+device object using the ``device`` (eg. true or false).
+
+.. code-block:: text
+
+   GET /api/v1/controller/group/?device={device}
```

Comment:
This in the admin is called `empty=`, can we maintain consistency please?

Reply:
sure